common/src/main/java/com/microsoft/identity/common/SharedPreferencesHelper.java

```diff
@@ -0,0 +1,90 @@
+package com.microsoft.identity.common;
+
+import android.content.Context;
+import android.content.SharedPreferences;
+
+/**
+ * Convenience class for accessing {@link SharedPreferences}.
+ */
+public class SharedPreferencesHelper {
```

Comment:
Should we call this SharedPreferenceSet? Or SharedPreferenceSetManager?  SharedPreferences docs refer to managing a "set" of preferences.  When created this object will allow the management of a set of preferences and doesn't allow the management of other preferences.  

Also do we want to have a property of this object indicating the name of the shared preference set that this class manages?

Reply:
I think I dislike `Set` in the name as that [to my eyes] implies it implements [Collection>Set](https://developer.android.com/reference/java/util/Set.html) -- how about just `SharedPreferencesManager` ? Maybe another alternative?

Reply:
SharedPreferencesFile or SharedPreferencesFileManager?

Reply:
Sounds great - I'll go with the latter 😄